src/lib/__tests__/insights-client-test.ts

```diff
@@ -102,7 +123,14 @@ describe('withInsights', () => {
       createWidgetWithInsights({ renderFn, instantSearchInstance, results });
 
       const [renderProps] = renderFn.mock.calls[0];
-      expect(renderProps).not.toHaveProperty('insights');
+      expect(() => {
+        renderProps.insights('clickedObjectIDsAfterSearch', {
+          eventName: 'add to favorites',
+          objectIDs: ['1'],
+        });
+      }).toThrowErrorMatchingInlineSnapshot(
+        `"The \`insightsClient\` option has not been provided to \`instantsearch\`."`
```

Comment:
What if we also provide a url to the doc? Too much?

https://www.algolia.com/doc/guides/getting-insights-and-analytics/search-analytics/click-through-and-conversions/how-to/send-click-and-conversion-events-with-instantsearch/js/

(btw, that's a one long url 😅)

Reply:
Good call

Reply:
Actually I added a URL to instantsearch docs. it's a generated URL and just as informative.